src/modules/dom.js

```diff
@@ -0,0 +1,32 @@
+import { url } from './url.js';
+import { getTotalMeal } from './mealcounter.js';
+
+const content = document.getElementById('content');
+function show(meal) {
```

Comment:
- [ ]  as part of the requirement, you guys should stick with ES6 as you did with most of your functions, kindly update function `show` to be an arrow function 🙏